components/constants.jsx

```diff
@@ -0,0 +1 @@
+export const STRING_DESC_LIMIT = 10;

```

Comment:
I'm not sure how I feel about a constants file. I might be fine with it if the names of the constants give enough context about where they will be used.

Reply:
Would you prefer magic numbers around files instead? I'll add more context to the constants. The reason I used constants file is because I plan on using these variables in multiple files. In the future, if a constant is only used in that file, I'll just have it in that file.